patches/chrome-browser-search_engines-template_url_service_factory.cc.patch

```diff
@@ -1,17 +1,13 @@
 diff --git a/chrome/browser/search_engines/template_url_service_factory.cc b/chrome/browser/search_engines/template_url_service_factory.cc
-index b3f207c4d728b68fa725654e916f0d8d1e3b0c49..d93ca1e804bb0b7a83def669fa66c2ae6289982a 100644
+index b3f207c4d728b68fa725654e916f0d8d1e3b0c49..58d55a22ca188cb7a6c19cb49654c76be9094ffd 100644
 --- a/chrome/browser/search_engines/template_url_service_factory.cc
 +++ b/chrome/browser/search_engines/template_url_service_factory.cc
-@@ -82,7 +82,12 @@ void TemplateURLServiceFactory::RegisterProfilePrefs(
+@@ -82,6 +82,8 @@ void TemplateURLServiceFactory::RegisterProfilePrefs(
  
  content::BrowserContext* TemplateURLServiceFactory::GetBrowserContextToUse(
      content::BrowserContext* context) const {
-+#if defined(BRAVE_CHROMIUM_BUILD)
```

Comment:
chromium_src

Reply:
Ditto (re: doing this in a follow-up PR)

Reply:
Tracked in https://github.com/brave/brave-core/pull/7980